src/ImathTest/CMakeLists.txt

```diff
@@ -38,10 +38,32 @@ add_executable(ImathTest
 )
 
 target_link_libraries(ImathTest Imath::Imath)
+#target_compile_options(ImathTest PRIVATE -mavx2 -mf16c)
```

Comment:
is this handled elsewhere, or was it meant to be guarded with an IMATH_ option?

Reply:
well, no - this is sort of what I was discussing w/ Larry before - it doesn't actually affect any compiled code in the libImath.so for installing - but will change the implementations for testing purposes, so I was manually doing that. But we need to be running on a machine that has those instructions, which I'm not sure we can do (and I don't know how to do good cross-compiler/platform options like that), and so should probably be removed from the cmake file entirely. Ultimately, it's up to the user of half.h to be compiling with those options, at which point we use the f16c instructions. I have been trying to find out the hardware specs for the github actions runner nodes to see if I can define a test variant for that code path or not (the other code paths I can test with a #define at the top, which I do in the c code test)